workflows/MODEL_TO_DATA.nf

```diff
@@ -3,11 +3,11 @@
 nextflow.enable.dsl = 2
 
 // Empty string default to avoid warning
-params.submissions = ""
+params.submissions = "9743078"
```

Comment:
This needs to remain an empty string or it will never pick up on manifests. It is kinda weird but it needs to have a placeholder default value of ""